R/Ostats_multivariate_plot.R

```diff
@@ -1,11 +1,11 @@
-#' Plot multivariate community overlap
+#' Plot multivariate overlap
```

Comment:
Again I still do not see why we should delete the word community. It is confusing without. It is good to clarify later that it may be a population and not a community, but I still think it is good to generally say this is for community ecology analyses.

Reply:
I added community back in.